### PR TITLE
Corrections to Classification ROC, Docs

### PR DESCRIPTION
**Pull request by atavory**
Corrections to classifsication metrics:

1. ROC uses fall-out, not miss-rate (which is wrong)
2. Miss-rate bugfix
3. Doc imrovements

### CODE REVIEW

**Comment by atavory** [comment]
@tdcmeehan  Could you please review it? Many thanks!

File: presto-bytecode/src/test/java/com/facebook/presto/bytecode/TestClassGenerator.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void testGenerator()
             throws Exception
     {
         ClassDefinition classDefinition = new ClassDefinition(
-                a(PUBLIC, FINAL),
```
Comment (highker) [inline presto-bytecode/src/test/java/com/facebook/presto/bytecode/TestClassGenerator.java]: looks like a typo?

Comment (atavory) [reply presto-bytecode/src/test/java/com/facebook/presto/bytecode/TestClassGenerator.java]: Thanks! Not sure how it got there :-)

**Comment by tdcmeehan** [comment]
@atavory can you rebase on master?  It looks like this contains all of the original code and not the fix itself, which should be just a few lines.

**Comment by atavory** [comment]
Screwed up rebase. Starting from scratch.

**Pull request closed without merging**